src/websql/models/address.ts

```diff
@@ -0,0 +1,21 @@
+import IModel from './model';
+
+enum BIPType {
+  BIP32 = 0,
+  BIP44 = 1
+}
+export default interface IAddress extends IModel {
+  address: string;
+  walletId: string;
+  coinType: string;
+  chainIndex: number;
+  addressIndex: number;
+  /**
+   * As a part of BIP141, a superior efficient Tx format Segwit was introduced.
```

Comment:
don't do this.. don't put comment in middle, 
during interface description, describe all params and attach related notes